### PR TITLE
[T4A3][W09-A2]Shawn Lin Jingjue

### PR DESCRIPTION
**Pull request by e0003823**
ready for review

### CODE REVIEW

**Comment by bqnguyen94** [comment]
Reviewed during tutorial

File: src/seedu/addressbook/data/person/Block.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Block.java:3]: No header comment.

File: src/seedu/addressbook/data/person/Block.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+	public static final String EXAMPLE = "123";
+	public static final String MESSAGE_BLOCK_CONSTRAINTS = "Block numbers should only contain numbers and letters.";
+	//	    public static final String BLOCK_VALIDATION_REGEX = "\\d+";
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Block.java:7]: You can remove unused code as we you RCS.

File: src/seedu/addressbook/data/person/Block.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+	/**
+	 * @return the example
+	 */
+	public static String getExample() {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Block.java:11]: Better use method description for trivial method, like `Returns ...`. BTW, the method name is not clear.

File: src/seedu/addressbook/data/person/Contact.java [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+public class Contact {
+	  	private String example = "";
+	    private String message_constraints = "";
+	    private String validation_regex = "";
+	    private String value = "";
+	    private boolean isPrivate;
+	    
+	    public Contact(String example, String message_constraints, String validation_regex, String value,
+		        boolean isPrivate) throws IllegalValueException {
+			super();
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Contact.java:14]: I don't think you need this.

File: src/seedu/addressbook/data/person/Contact.java [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+			this.example = example;
+			this.message_constraints = message_constraints;
+			this.validation_regex = validation_regex;
+			this.isPrivate = isPrivate;
+			
+			String trimmedContact = value.trim();
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Contact.java:20]: Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)

**Comment by jeffryhartanto** [comment]
@e0003823 
Some comments added. Please close the PR after reading comments. Make sure next time you submit clean PR. :)

**Pull request closed without merging**